prime-router/metadata/hl7_mapping/common/specimen.yml

```diff
@@ -0,0 +1,33 @@
+constants:
+  hl7SegmentGroup: /ORDER/OBSERVATION_REQUEST/SPECIMEN/SPM
+elements:
+  - name: specimen-set-id
+    value: [ '"1"' ]
+    hl7Spec: [ '%{hl7SegmentGroup}-1' ]
+
+  - name: specimen-type
+    resource: '%resource.type'
+    constants:
+      cweFieldPath: '%{hl7SegmentGroup}-4'
+    schema: ../common/datatype/ce-coded-with-exceptions
+
+  - name: specimen-collection-date/time
+    condition: '%resource.collection.collected.exists() and %resource.collection.collected is dateTime'
+    value: [ '%resource.collection.collected' ]
+    hl7Spec: [ '%{hl7SegmentGroup}-17' ]
+
+  - name: specimen-collection-time
+    condition: '%resource.collection.collected.exists() and %resource.collection.collected is dateTime'
+    value: [ '%resource.collection.collected' ]
+    hl7Spec: [ '%{hl7SegmentGroup}-17-1' ]
+
```

Comment:
Why do both `specimen-collection-date/time` and `specimen-collection-time` exist with the same `condition` and `value`?

Reply:
I deleted one entry, good catch!